<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionsWindowThree</class>
 <widget class="QDialog" name="QuestionsWindowThree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>741</width>
         <height>451</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>330</y>
          <width>721</width>
          <height>36</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="nextButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>410</y>
          <width>92</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QPushButton" name="previousButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>92</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>40</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="questionPrompt">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>701</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="finishButton">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>410</y>
          <width>92</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>40</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Time remaining:</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
